app/src/main/java/at/roteskreuz/stopcorona/screens/dashboard/changelog/ChangelogController.kt

```diff
@@ -0,0 +1,54 @@
+package at.roteskreuz.stopcorona.screens.dashboard.changelog
+
+import android.content.Context
+import android.text.SpannableString
+import at.roteskreuz.stopcorona.model.manager.Changelog
+import at.roteskreuz.stopcorona.screens.base.epoxy.buttons.buttonType1
+import at.roteskreuz.stopcorona.screens.base.epoxy.copyText
+import at.roteskreuz.stopcorona.screens.base.epoxy.emptySpace
+import at.roteskreuz.stopcorona.screens.base.epoxy.headlineH1
+import at.roteskreuz.stopcorona.screens.base.epoxy.image
+import at.roteskreuz.stopcorona.skeleton.core.utils.adapterProperty
+import com.airbnb.epoxy.EpoxyController
+
+class ChangelogController(
+    private val context: Context,
+    private val onCtaClick: () -> Unit
+) : EpoxyController() {
```

Comment:
Just for info, you can use `TypedEpoxyController< Changelog>` when you don't want to draw something else than the loaded value. You can keep it.

Reply:
good catch